packages/cubejs-server-core/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "include": [
     "src",
     "test"
-  ],
+, "../cubejs-backend-shared/src/logger.js"  ],
```

Comment:
Weird. Do you have a problem with `yarn link`?

Reply:
Ehum. Not sure what automatic something did that :D